src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -28,56 +24,106 @@ export interface ShareButtonProps {
   showEmbedButton?: boolean;
 }
 
+interface DialogProps {
+  open: boolean;
+  closeDialog: () => void;
+  openDialog: () => void;
+  modalContent: MetricModalContent;
+  modalHeader: string;
+}
+
+const ShareButtonBlock: React.FC<ShareButtonProps> = ({
+  ...shareButtonProps
+}) => {
+  return (
+    <ButtonContainer>
+      <ShareButtons {...shareButtonProps} />
+    </ButtonContainer>
+  );
+};
```

Comment:
Not sure if we need this component, is really small 😃

Reply:
yeah, its just repeated twice down in the main component, so i pulled it out into its own thing